src/app/components/engangsstonad/steps/Engangsstonad.step2.js

```diff
@@ -31,32 +47,104 @@ class Step2 extends Component {
                     </Normaltekst>
                 </DialogBox>
                 <Element>Søknaden gjelder en fødsel som er...</Element>
-                <ToggleGruppe onChange={this.toggleBirth} name="isChildBorn">
-                    <ToggleKnapp value="childBorn">tilbake i tid</ToggleKnapp>
-                    <ToggleKnapp value="childNotBorn">frem i tid</ToggleKnapp>
+                <ToggleGruppe onChange={this.props.toggleChildBorn} name="isChildBorn">
+                    <ToggleKnapp
+                        defaultChecked={this.props.childBorn === 'childBorn'}
+                        value="childBorn"
+                    >
+                        tilbake i tid
+                    </ToggleKnapp>
+                    <ToggleKnapp
+                        defaultChecked={this.props.childBorn === 'childNotBorn'}
+                        value="childNotBorn"
+                    >
+                        frem i tid
+                    </ToggleKnapp>
                 </ToggleGruppe>
-                {this.state.childBorn === false &&
+                {this.props.childBorn === 'childNotBorn' &&
                     <div>
                         <Element>og jeg venter...</Element>
-                        <ToggleGruppe onChange={() => undefined} name="noOfChildren">
-                            <ToggleKnapp value="1">et barn</ToggleKnapp>
-                            <ToggleKnapp value="2">tvillinger</ToggleKnapp>
-                            <ToggleKnapp value="3">trillinger</ToggleKnapp>
+                        <ToggleGruppe onChange={this.props.toggleNoOfChildren} name="noOfChildren">
+                            <ToggleKnapp
+                                defaultChecked={this.props.noOfChildren === '1'}
```

Comment:
noOfChildren kunne kanskje vært en vanlig int?

Reply:
ToggleGruppen her skal egentlig være 1, 2 eller flere som vi for tiden ikke har et designforslag på, så det er bare alternativ 1 og 2 som skal mappes direkte til verdi. Kan sikkert være int eller parses til int.